packages/logs/lib/es/schema.ts

```diff
@@ -68,7 +95,7 @@ const props: Record<keyof MessageRow, opensearchtypes.MappingProperty> = {
 };
 
 export const indexMessages: opensearchtypes.IndicesCreateRequest = {
-    index: envs.NANGO_LOGS_OS_INDEX ?? 'messages',
+    index: `20240522_${envs.NANGO_LOGS_OS_INDEX ?? 'messages'}`,
```

Comment:
What is this `20240522_` prefix?

Reply:
just versionning to avoid the migration of the data in staging